src/ResourceManager/Insights/Commands.Insights/Diagnostics/SetAzureRmDiagnosticSettingCommand.cs

```diff
@@ -31,6 +32,11 @@ namespace Microsoft.Azure.Commands.Insights.Diagnostics
     [Cmdlet(VerbsCommon.Set, "AzureRmDiagnosticSetting"), OutputType(typeof(PSServiceDiagnosticSettings))]
```

Comment:
Change cmdlet name to update and place an alias to Set (or vice-versa)  Also, please note in upcoming breaking changes that the cmdlet name is changing to update: https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Insights/documentation/upcoming-breaking-changes.md